zwave_js_server/model/node.py

```diff
@@ -383,6 +383,21 @@ async def async_refresh_info(self) -> None:
         """Send refreshInfo command to Node."""
         await self.async_send_command("refresh_info", wait_for_result=False)
 
+    async def async_refresh_values(self) -> None:
+        """Send refreshValues command to Node."""
+        await self.async_send_command(
+            "refresh_values", wait_for_result=False, require_schema=4
+        )
+
+    async def async_refresh_cc_values(self, command_class: int) -> None:
```

Comment:
```suggestion
    async def async_refresh_cc_values(self, command_class: CommandClass) -> None:
```
Is `CommandClass` a better type?

Reply:
@MartinHjelmare thoughts on this? Will this work cleanly if someone passes an integer or the enum

Reply:
answered my own question, yes it does

Reply:
thanks @kpine !